Steepfile

```diff
@@ -0,0 +1,17 @@
+D = Steep::Diagnostic
+
+target :lib do
+  signature "sig"
+  check "lib"
+  configure_code_diagnostics(D::Ruby.default)
+  library "pathname"
+  library "date"
+end
+
+target :test do
```

Comment:
Looks like there is no `test` directory in this repo?
Supporting rspec is difficult. So, I don't think having Steep enabled with specs doesn't make sense...

Reply:
Goodness, that's embarrassing! Never came back to update this after I understood how the config worked -.-